### ISSUE TITLE
Missing source files

### ISSUE THREAD

**Issue by jgk1981** [microsoft/Microsoft-Rocketbox#21]
Many of the animals are mising from the source folders as i see. There i see only the  image of the model but not the .max file.
If can update it will be great. Thank you

**Comment by margonzalezfranco** [comment]
Hi, yes I see this problem. I have lost all my access to edit this repo since my departure from Microsoft

Best you can do is go to the assets/animals/../export folder and reverse the fbx into a max.

I have checked and the fbxs seem to all be there.

thanks

**Comment by jgk1981** [comment]
Yes. will recreate the skeleton if need from the fbx files. Good thing that all is biped so will be fairly easy process. Thanks
